src/fsharp/ast.fs

```diff
@@ -1475,11 +1502,11 @@ type QualifiedNameOfFile =
 
 [<NoEquality; NoComparison>]
 type ParsedImplFileInput = 
-    | ParsedImplFileInput of string * (*isScript: *) bool * QualifiedNameOfFile * ScopedPragma list * ParsedHashDirective list * SynModuleOrNamespace list * (bool * bool)
+    | ParsedImplFileInput of fileName:string * isScript:bool * QualifiedNameOfFile * ScopedPragma list * ParsedHashDirective list * SynModuleOrNamespace list * (bool * bool)
```

Comment:
They indicate ``(isLastCompiland, isExe)`` - last file in input, and are we treating the input as if it's defining a .EXE (no warning on missing "module" in last compiland)